dl_lib/layers/ROIAlign/ROIAlign_cuda.cu

```diff
@@ -334,7 +338,7 @@ at::Tensor ROIAlign_forward_cuda(
   auto output_size = num_rois * pooled_height * pooled_width * channels;
   cudaStream_t stream = at::cuda::getCurrentCUDAStream();
 
-  dim3 grid(std::min(at::cuda::ATenCeilDiv(output_size, 512L), 4096L));
+  dim3 grid(std::min(ceil_div((int)output_size, 512), 4096));
```

Comment:
at::cuda::ATenCeilDiv works for all platform, the real reason for not working on windows is 'L'

Reply:
I will change it and try to recompile.

Reply:
If i remove "L", could this function run correctly on linux? Could i just simple "L"?